src/magic_folder/snapshot.py

```diff
@@ -164,6 +164,58 @@ def _get_synchronous_content(self):
         with open(self.content_path, "rb") as f:
             return f.read()
 
+    def serialize(self):
+        """
```

Comment:
Since we used `to_json()` elsewhere for "serialize to JSON" maybe "to_json()" and "from_json()" for these methods? Kind of a "bikeshed" comment :/